src/command.rs

```diff
@@ -26,27 +26,28 @@ use crate::{generator, generator::Language};
 use std::path::PathBuf;
 
 use async_recursion::async_recursion;
-use eyre::{ensure, Context, ContextCompat};
+use eyre::Context;
```

Comment:
```suggestion
use eyre::{Context, ContextCompat};
```

Reply:
I think this is worse, `eyre` only provides `ContextCompat` for helping migrate existing `anyhow` projects, but [they recommend not using it in general](https://docs.rs/eyre/latest/eyre/#context-and-option).